lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -197,7 +197,64 @@ do {
         weak_self ? "Swift.Optional where Wrapped == " : "";
 
     // The expression text is inserted into the body of $__lldb_wrapped_expr_%u.
-    wrapped_stream.Printf(R"(
+    if (options.GetEvaluateSelfAsGeneric()) {
+      // When evaluating as generic, we have to bypass the Swift typesystem.
```

Comment:
Can you use more words for this sentence? The rest of the comment explains really well *what* happens, but I think we could use more words explaining the *why*.